### ISSUE TITLE
Mailinglist instead of RC #support or github for user based discussion

### ISSUE THREAD

**Issue by localguru** [RocketChat/Rocket.Chat#6578]
Hi,

I use RC since a month and posted some issues with close to no response in the #support channel nor on github. I know that one can't expect an answer to any question, but I think the response proportionately to the success of RC and to the user base (11.500 stars here on github) is quite low. To my mind the #support channel is too fleetingly, users come and go and a asynchronous communication seems to be difficult. Github on the other side could do this job, but most people see it as a bugtracker not as a communication/support platform. Therefore: what about a classic mailinglist for user based discussion? I would help to support it technically. @RC team: please don't understand this as criticism. These are just some nightly thoughts to push the rocket higher :)

Ciao
Marcus

**Comment by jszaszvari** [comment]
Sorry but who uses mailing lists these days? I know they are still around but just because every message lands in someone's inbox doesent mean people are going to respond. 

Stay in the support channel and make sure you have email notifications turned on. 

Between here and the support channel there is more than enough. Depending on your issue make sure you search through the closed issues on GitHub. 

If you don't get a response on chat or here, I'm not sure what makes you think someone's going to respond on a mailing list. 

For people interested in a mailing list they can subscribe to the issues feed here and recieve a email notification for every issue and comment posted, and even reply via email. It's basically the same

**Comment by localguru** [comment]
@jszaszvari All big open source projects offering mailing lists in "these days", still. ;) The difference between a mail list and a channel is, that mailing lists are more efficient for deferred communications, content is archived and thread based! A mail client has more options than a channel. The nature of a channel is more or less a just in time communication; after a while it's hard to follow a thread. It might be my personal preference, but a channel has not a good visibility, content is just flying through. The notifications on github is not the same as a mailing list. If you watch a project, you'll be informed not only about issues, but about pull request too. Following each issue is not easy to handle.

**Comment by localguru** [comment]
Filter github messages by Message-ID seems to be a way to sort "watched" messages e.g. by

 RocketChat/Rocket.Chat/releases/
 RocketChat/Rocket.Chat/issues/
 RocketChat/Rocket.Chat/pull/

Ciao!

**Comment by geekgonecrazy** [comment]
https://forums.rocket.chat
